src/test/java/com/o19s/es/explore/StatisticsHelperTests.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.o19s.es.explore;
+
+import org.apache.lucene.util.LuceneTestCase;
+import org.junit.Test;
+
+public class StatisticsHelperTests extends LuceneTestCase {
+    private final float[] dataset = new float[] {
+      0.0f, -5.0f, 10.0f, 5.0f
+    };
+
+    @Test(expected=AssertionError.class)
```

Comment:
forbidden api will complain here, junit anotations are not allowed.
also you don't really need to test this, inline assertion are meant to be tested alongside normal tests and will blow-up if something went wrong.